apps/src/assetManagement/AssetManager.jsx

```diff
@@ -135,7 +135,7 @@ module.exports = React.createClass({
 
       assetList = (
         <div>
-          <div style={{maxHeight: '330px', overflowY: 'scroll', margin: '1em 0', paddingRight: '15px'}}>
+          <div style={{maxHeight: '330px', overflowY: 'scroll', margin: '1em 0'}}>
```

Comment:
Punting on this for now since this is part of an existing component.  I've extracted the styles in the new `IconLibrary` and child components.